ortools/xpress/parse_header_xpress.py

```diff
@@ -172,12 +160,34 @@ def parse(self, filepath):
         with open(filepath) as fp:
             all_lines = fp.read()
 
+        XPRSprob_section = False
+
         for line in all_lines.splitlines():
             if not line:  # Ignore empty lines.
                 continue
-            if re.match(r'/\*', line, re.M):  # Ignore comments.
+
+            if " * control parameters for XPRSprob" in line:
+                XPRSprob_section = True
                 continue
 
+            if XPRSprob_section and "/***************************************************************************\\" in line:
+                XPRSprob_section = False
```

Comment:
A bit weird, care to explain ?

Reply:
we only handle XPRESS object XPRSprob, so we only have to import its control parameters. the xprs.h header file contains control parameters for XPRSprob but also for XPRSmipsolpool, XPRSmipsolenum.. this variable detects the XPRSprob sections by parsing comments and detecting sections (delimited by "*****"..)

Reply:
I added a comment in the new method